CMakeLists.txt

```diff
@@ -4,9 +4,9 @@ include(CheckCXXCompilerFlag)
 include(cmake/gotcha.cmake)
 CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
 
-set(LIBTOOL_INTERFACE 2)
+set(LIBTOOL_INTERFACE 1)
 set(LIBTOOL_REVISION 0)
-set(LIBTOOL_AGE 2)
+set(LIBTOOL_AGE 4)
```

Comment:
libtool interface versions should never go backwards, note "version" != "library interface"

Reply:
The problem is this create gotcha library with different version than that on documentation. I will decouple that. Thank you for bringing this to my attention.

Reply:
I am fixing this on #115